.github/workflows/npm-release-open-source.yml

```
@@ -0,0 +1,57 @@
+name: Reusable WF - Release NPM Package
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: The package version of the release
+        type: string
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+# Permissions expanded to allow for provenance
+permissions:
+  contents: read
```

The GitHub release action necessitates that this be:

```suggestion
  contents: write
```